test/mustachio/builder_test.dart

```diff
@@ -135,7 +135,18 @@ class Bar {}
       expect(generatedContent, contains('''
         'b1': Property(
           getValue: (CT_ c) => c.b1,
-          getProperties: _Renderer_bool.propertyMap,
+          renderVariable:
+              (CT_ c, Property<CT_> self, List<String> remainingNames) {
+            if (remainingNames.isEmpty) return self.getValue(c).toString();
+            var name = remainingNames.first;
+            if (_Renderer_bool.propertyMap().containsKey(name)) {
+              var nextProperty = _Renderer_bool.propertyMap()[name];
+              return nextProperty.renderVariable(
+                  self.getValue(c), nextProperty, [...remainingNames.skip(1)]);
+            } else {
+              throw MustachioResolutionError();
```

Comment:
I wonder if it is worth it to include more detail if we throw here; as you're the one debugging right now you're in the best position to know.  Definitely fine for now.

Reply:
Yeah, at this position, we don't know the whole string of names that form the key. Additionally, my little parser does not write SourceSpans. 😱 😱 😱 This is probably important. :/

In any case, we catch this error and throw a new one with the right details.